.travis.yml

```diff
@@ -18,7 +18,7 @@
 # http://docs.travis-ci.com/user/customizing-the-build/
```

Comment:
Hmm... Can you move changes in this file to a separate commit/pull-request? To the moment we actually use development server on Travis, e.g., during integration tests. Because right now this code is not used, and essentially just makes CI runs slower.